package/debian/DEBIAN/control

```diff
@@ -1,8 +1,8 @@
 Package: easee
-Version: 0.0.9
+Version: 2.0.0-rc.3
```

Comment:
We will prolly decide to skip rc's in 30 minutes :P

Reply:
leftover, will remove it :P